packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -43,6 +51,14 @@ export interface FieldChangeRebaser<TChangeset> {
         change: TaggedChange<TChangeset>,
         invertChild: NodeChangeInverter,
         genId: IdAllocator,
+        crossFieldManager: CrossFieldManager,
```

Comment:
can we make this `crossFieldManager: CrossFieldManager<NodeChangeset>` to save the implementers from casting?
(same on the other functions)

Reply:
The CrossFieldManager stores arbitrary data for each key, so the fields have to cast it to the kind of data they expect.